plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -2817,12 +2824,72 @@ func TestInterPodAffinityWithMultipleNodes(t *testing.T) {
 			},
 			test: "NodeA and nodeB have same topologyKey and label value. NodeA has an existing pod that match the inter pod affinity rule. The pod can not be scheduled onto nodeA and nodeB but can be schedulerd onto nodeC",
 		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: v1.ObjectMeta{
+					Labels: map[string]string{"foo": "123"},
+					Annotations: map[string]string{
+						v1.AffinityAnnotationKey: `
```

Comment:
wow, yes; and I think we should also remove `v1.AffinityAnnotationKey` from the code.